output/slide-deck.qmd

```diff
@@ -66,7 +66,24 @@ knitr: true
 
 -   <https://unedo365-my.sharepoint.com/:u:/g/personal/dvmorillo_psi_uned_es/EZBXy-Z5U75FsCKM_P0lTTkBzx3eH0IkrczYzmpPiCX31g?e=4ko0WX>
```

Comment:
This will be a shortened "bit.ly" link in the final version, so all the "Setup" will fit in a single slide (with a few tweaks, see suggestions below)